code/pizzabot.js

```diff
@@ -1,7 +1,193 @@
+
+- 1-2 pizzas: `The pizzas will take 10 minutes.`
+- 3-5 pizzas: `The pizzas will take 15 minutes.`
+- 6+ pizzas: `The pizzas will take 20 minutes.`
+
+For example, if I ordered 3 Hawaiian pizzas, the message should now read:
+"Great, I'll get started on your Hawaiian Pizza right away, it will cost 240 kr. The pizzas will take 15 minutes."
+
+*/
+
+/*
+const X = "Vegetarian Pizza"
+const Y = "Hawaiian Pizza"
+const Z = "Pepperoni Pizza"
+const pizzaPrice = 80
+
+/*
+alert(`Hey! Happy to serve your pizza. On our menu we have ${X}, ${Y} and ${Z}`)
+
+let orderName = prompt(`Enter the name of the pizza you want to order today`)
+
+if (orderName == X || orderName == Y || orderName == Z) {
+  const orderQuantity = prompt(`"How many of ${orderName} do you want?`)
+  const orderTotal = (pizzaPrice * orderQuantity)
+  if (orderQuantity < 3) {
+    alert(`The pizzas will take 10 minutes. Great, I'll get started on your ${orderName} right away, it will cost ${orderTotal} kr.`)
+  } else if (orderQuantity >= 3 && orderQuantity <= 5) {
+    alert(`The pizzas will take 15 minutes. Great, I'll get started on your ${orderName} right away, it will cost ${orderTotal} kr.`)
+  } else {
+    alert(`The pizzas will take 20 minutes. Great, I'll get started on your ${orderName} right away, it will cost ${orderTotal} kr.`)
+  }
+} else {
+  alert(`Select a pizza from the menu`)
+}
+
+
+
+/*
+
+## **Iteration 6 - Add functions**
+
+Clean up your code by moving some code into functions and then calling that function when needed.
+
+The functions should all reflect the tasks we've completed so far. So, the functions should be:
+
+1. `checkOrderName()` which should take the `orderName` variable as an argument and return `true` or `false` if the pizza is on the menu or not.
+2. `totalCost()` which takes `orderQuantity` as an argument and returns the total cost for the order.
+3. `cookingTime()` which takes `orderQuantity` and returns the number of minutes it will take to finish the order.
+
+
+*/
+
+
+const X = 'Vegetarian Pizza';
+const Y = 'Hawaiian Pizza';
+const Z = 'Pepperoni Pizza';
+const pizzaPrice = 80;
+
+
+// function for check if order exist on menu
+const checkOrderName = (orderName) => {
+  if (orderName == X || orderName == Y || orderName == Z) {
+    console.log(true);
+    return true;
+  } else {
+    console.log(false);
+    return false;
+  }
+};
+
+// function for total cost of order
+const totalCost = orderQuantity => {
+  const orderTotal = (pizzaPrice * orderQuantity);
+  console.log(orderTotal);
+  return orderTotal;
+};
+
+//function cooking time
+const cookingTime = orderQuantity => {
+  if (orderQuantity < 3) {
+    console.log(10);
+    return 10;
+  } else if (orderQuantity >= 3 && orderQuantity <= 5) {
+    console.log(15);
+    return 15;
+  } else {
+    console.log(20);
+    return 20;
+  }
+};
+
+//function for order summary
+const orderDetails = (onMenu, orderName) => {
+  if (onMenu == true) {
```

Comment:
This checks if its true aswell! 
```suggestion
  if (onMenu) {
```